eclass/virtualx.eclass

```diff
@@ -231,44 +183,4 @@ virtx() {
 	return 0 # always return 0, it can be altered by failed kill for Xvfb
 }
 
-# @FUNCTION: Xmake
-# @DESCRIPTION:
-# Same as "make", but set up the Xvfb hack if needed.
-# Deprecated call.
-Xmake() {
-	debug-print-function ${FUNCNAME} "$@"
-
-	[[ ${EAPI} == [45] ]] \
-		|| die "${FUNCNAME} is unsupported in EAPI > 5, please use 'virtx emake -j1 ....'"
-
-	eqawarn "you should not execute make directly"
-	eqawarn "rather execute Xemake -j1 if you have issues with parallel make"
-	VIRTUALX_COMMAND="emake -j1" virtualmake "$@"
-}
-
-# @FUNCTION: Xemake
-# @DESCRIPTION:
-# Same as "emake", but set up the Xvfb hack if needed.
-Xemake() {
```

Comment:
`Xemake`, unlike `Xmake` and `Xeconf` actually still seems to be used in tree :(

Reply:
kept them as stub functions that unconditionally die.